app/src/shared/test/components/Nav/nav.test.jsx

```diff
@@ -0,0 +1,184 @@
+import React from 'react'
```

Comment:
Added a "Data Unions" section to main menu, this tests that it won't be shown without the flag.